static/app/views/organizationStats/usageTable/index.tsx

```diff
@@ -143,3 +164,15 @@ const ErrorMessages = styled('div')`
   margin-top: ${space(1)};
   font-size: ${p => p.theme.fontSizeSmall};
 `;
+
+const Title = styled('strong')`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  margin-bottom: ${space(1)};
+`;
+
+const Description = styled('span')`
```

Comment:
We actually have a component called EmptyMessage which you can probably use :)

https://storybook.sentry.dev/?path=/story/layouts-emptystate-emptymessage--in-panel-with-title-and-description

Reply:
ooo looks good, ive replaced what i had with EmptyMessage